File: app/src/main/java/bj/discogsbrowser/model/order/Item.java

package bj.discogsbrowser.model.order;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import bj.discogsbrowser.model.listing.Release;

public class Item
{

    @SerializedName("release")
    @Expose
    private Release release;
    @SerializedName("price")
    @Expose
    private Price price;
    @SerializedName("id")
    @Expose
    private Integer id;

    public Release getRelease()
    {
        return release;
    }

    public void setRelease(Release release)
    {
        this.release = release;
    }

    public Price getPrice()
    {
        return price;
    }

    public void setPrice(Price price)
    {
        this.price = price;
    }

    public Integer getId()
    {
        return id;
    }

    public void setId(Integer id)
    {
        this.id = id;
    }

}

File: app/src/main/java/bj/discogsbrowser/model/order/Item.kt
package bj.discogsbrowser.model.order

import bj.discogsbrowser.model.common.Price
import bj.discogsbrowser.model.listing.ListingRelease

/**
 * Created by Josh Laird on 19/05/2017.
 */
data class Item(val release: ListingRelease, val price: Price, val id: String)